src/settings/toml/tests/deployments.rs

```diff
@@ -94,58 +109,147 @@ fn it_errors_on_zoneless_deploy_config_workers_dev_false() {
     let manifest = Manifest::from_str(&toml_string).unwrap();
 
-    let environment = None;
-    let actual_deploy_config = manifest.deploy_config(environment).unwrap();
     let expected_routes = vec![Route {
         script: Some(script_name.to_string()),
         pattern: PATTERN.to_string(),
         id: None,
     }];
-    let expected_deploy_config = DeployConfig::Zoned(Zoned {
+    let expected_deployments = vec![DeployTarget::Zoned(ZonedTarget {
         zone_id: ZONE_ID.to_string(),
         routes: expected_routes,
+    })];
+    let environment = None;
+    let actual_deployments = manifest.get_deployments(environment).unwrap();
+
+    assert_eq!(actual_deployments, expected_deployments);
+}
+
+#[test]
+fn it_can_get_a_scheduled_no_workers_dev_no_zoned() {
+    let script_name = "single_schedule";
+
+    let crons = vec!["0 * * * *".to_owned()];
+
+    let mut test_toml = WranglerToml::webpack(script_name);
+    test_toml.account_id = Some(ACCOUNT_ID);
+    test_toml.triggers = Some(Triggers {
+        crons: Some(crons.clone()),
     });
 
-    assert_eq!(actual_deploy_config, expected_deploy_config);
+    let toml_string = toml::to_string(&test_toml).unwrap();
+    let manifest = Manifest::from_str(&toml_string).unwrap();
+
+    let expected_deployments = vec![DeployTarget::Schedule(ScheduleTarget {
+        account_id: ACCOUNT_ID.to_owned(),
+        script_name: script_name.to_owned(),
+        crons: crons,
+    })];
+    let environment = None;
+    let actual_deployments = manifest.get_deployments(environment).unwrap();
+
+    assert_eq!(actual_deployments, expected_deployments);
 }
 
 #[test]
-fn it_errors_on_single_route_deploy_config_empty_zone_id() {
+fn it_can_get_a_scheduled_in_env_no_workers_dev_no_zoned() {
+    let script_name = "single_schedule";
+
+    let env_crons = vec!["0 * * * *".to_owned()];
+
+    let env_config = EnvConfig {
+        triggers: Some(Triggers {
+            crons: Some(env_crons.clone()),
+        }),
+        ..EnvConfig::default()
+    };
+    let mut test_toml = WranglerToml {
+        account_id: Some(ACCOUNT_ID),
+        triggers: Some(Triggers {
+            crons: Some(vec!["0 * * * *".to_owned()]),
+        }),
+        ..WranglerToml::webpack(script_name)
+    };
+    test_toml
+        .env
+        .get_or_insert_with(Default::default)
+        .insert("b", env_config);
+
+    let toml_string = toml::to_string(&test_toml).unwrap();
+    let manifest = Manifest::from_str(&toml_string).unwrap();
+
+    let environment = Some("b");
+    let worker_name = manifest.worker_name(Some("b"));
+    let expected_deployments = vec![DeployTarget::Schedule(ScheduleTarget {
+        account_id: ACCOUNT_ID.to_owned(),
+        script_name: worker_name,
+        crons: env_crons,
+    })];
+    let actual_deployments = manifest.get_deployments(environment).unwrap();
+
+    assert_eq!(actual_deployments, expected_deployments);
+}
+#[test]
+fn it_errors_on_no_schedules_in_env() {
```

Comment:
should this be true? i think when we talked about this the assumption was that crons should be able to inherit.